unitree_go2/README.md

```diff
@@ -30,6 +30,18 @@ to OBJ format using [Blender](https://www.blender.org/).
    increased `impratio` to reduce slippage.
 8. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
 
+## MJX
+
+A version of the Unitree Go2 model for use in MJX is available in scene_mjx.xml with the following changes:
```

Comment:
Can you enclose all the variables such as `scene_mjx.xml` and `frictionloss` with ``?